### PR TITLE
Memstatus

### PR DESCRIPTION
**Pull request by robthomson**
-- Simple script that can run as a background task. -- it will print out memory status to the console every -- 5 seconds (or whatever interval you set)
-- this is helpfull when building memory sensitive scripts -- as it can aid tracking down memory intensive code.

### CODE REVIEW

File: lua/examples/memstatus/main.lua [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+function wakeup()
+
+    local now = os.clock()
+
+    if (now - wakeupScheduler) >= interval then
+        wakeupScheduler = now
+		
+		mem = system.getMemoryUsage()
```
Comment (bsongis) [inline lua/examples/memstatus/main.lua]: indentation issue

File: lua/examples/memstatus/main.lua [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+		local ramDiff = (mem.luaRamAvailable - prevluaRamAvailable)/1000 .. 'KB'
+		local bmpDiff = (mem.luaBitmapsRamAvailable - prevluaBitmapsRamAvailable)/1000 .. 'KB'
+
+		prevluaRamAvailable = mem.luaRamAvailable
+		prevluaBitmapsRamAvailable = mem.luaBitmapsRamAvailable
+		
+
+		print("-------------------------------------------------------") 
+		print("luaRamAvailable        : " .. mem.luaRamAvailable/1000 .. "KB" .. " ["..ramDiff.."]" )
```
Comment (bsongis) [inline lua/examples/memstatus/main.lua]: I think it should be "kB", not "KB"

File: lua/examples/memstatus/main.lua [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		local ramDiff = (mem.luaRamAvailable - prevluaRamAvailable)/1000 .. 'KB'
+		local bmpDiff = (mem.luaBitmapsRamAvailable - prevluaBitmapsRamAvailable)/1000 .. 'KB'
+
+		prevluaRamAvailable = mem.luaRamAvailable
+		prevluaBitmapsRamAvailable = mem.luaBitmapsRamAvailable
+		
+
+		print("-------------------------------------------------------") 
+		print("luaRamAvailable        : " .. mem.luaRamAvailable/1000 .. "KB" .. " ["..ramDiff.."]" )		
+		print("luaBitmapsRamAvailable : " .. mem.luaBitmapsRamAvailable/1000 .. "KB" .. " ["..bmpDiff.."]" )
```
Comment (bsongis) [inline lua/examples/memstatus/main.lua]: in english there is no space before `:` (but in french there is!)

Comment (bsongis) [reply lua/examples/memstatus/main.lua]: unless you want to align and then it should be `........`

File: lua/examples/memstatus/main.lua (hunk 1 above)
Comment (robthomson) [reply lua/examples/memstatus/main.lua]: Resolved by pushing script through lua-format

File: lua/examples/memstatus/main.lua (hunk 3 above)
Comment (robthomson) [reply lua/examples/memstatus/main.lua]: This is about aligning.. is your thought to print as:

print("luaRamAvailable............: " .. mem.luaRamAvailable/1000 )

File: lua/examples/memstatus/main.lua (hunk 2 above)
Comment (robthomson) [reply lua/examples/memstatus/main.lua]: Fixed in latest commit

File: lua/examples/memstatus/main.lua (hunk 3 above)
Comment (robthomson) [reply lua/examples/memstatus/main.lua]: Have applied commit on the .... basis

**Pull request merged**